maven-model/src/test/resources/xml/pom.xml

```diff
@@ -25,7 +25,7 @@ under the License.
   <parent>
     <groupId>org.apache.maven</groupId>
     <artifactId>maven</artifactId>
-    <version>4.0.0-alpha-1-SNAPSHOT</version>
+    <version>4.0.0-alpha-3-SNAPSHOT</version>
```

Comment:
This file is a test pom which is simply loaded, the version number is not important.  For clarity, if we really want to change it, maybe use something like `dummy-version` instead of something that can be misinterpreted as a real version.

Reply:
Yes.
The tricky part here is that in this file all versions were missed.
I'll have to check if something as resource filtering can handle this file.

Reply:
Resource filtering works.
Even though the version is `dummy-version` the places where it says `${project.version}` now actually becomes the actual version of the project for testing purposes.